scripts/check-asciidoctor-build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+# ensure asciidoctor is installed
+if ! command -v asciidoctor ;
+then
+    echo "Asciidoctor is not installed. Please install it and try again."
+    exit 127
+fi
+
+# get the *.adoc modules and assemblies in the pull request
+FILES=$(git diff-tree HEAD HEAD~1 --no-commit-id --name-only -r)
+
+# get the modules in the PR, search for assemblies that include them, and concat with any updated assemblies files
+get_updated_assemblies () {
+    MODULES=$(echo "$FILES" | awk '/modules\/(.*)\.adoc/')
+    # $UPDATED_ASSEMBLIES is the list of assemblies that contains changed modules
+    UPDATED_ASSEMBLIES=$(find . -path "./modules" -prune -o -name "*.adoc" -type f -exec grep -l "$MODULES" {} \; | sed 's|^\./||')
+    # $ASSEMBLIES is the list of modifed assemblies
+    ASSEMBLIES=$(echo "$FILES" | awk '!/modules\/(.*)\.adoc/' | awk '/(.*)\.adoc/')
+    # concatenate both lists and remove dupe entries
+    ALL_ASSEMBLIES=$(echo "$UPDATED_ASSEMBLIES $ASSEMBLIES" | tr ' ' '\n' | sort | uniq)
```

Comment:
**thought:** I used `sort -u` before so I was wondering about this one. No changes needed but a good read if you want to https://unix.stackexchange.com/questions/76049/what-is-the-difference-between-sort-u-and-sort-uniq